tests/unit/amo/sagas/test_collections.js

```diff
@@ -537,7 +537,7 @@ describe(__filename, () => {
         // Make sure the the collection is not deleted.
```

Comment:
This comment should be updated to read: `Make sure the the collection is not unloaded.`